core/templates/components/ck-editor-helpers/ck-editor-4-rte.component.ts

```diff
@@ -386,10 +386,6 @@ export class CkEditor4RteComponent implements AfterViewInit, OnChanges,
       (
         this.elementRef.nativeElement as HTMLElement
       ).setAttribute('style', 'display: block');
-      // Focus on the CK editor text box.
-      (
-        this.elementRef.nativeElement.children[0].children[1] as HTMLElement
-      ).focus();
```

Comment:
Why is this removed?

Reply:
This is removed because when ever any page/modal is being loaded which has has the ckeditor, then the focus directly jumps to text box -- resulting in unordered naviagtion for screen readers and keyboard users. To avoid this behaviour the has to be removed. Hope this answers your question. Thanks!

Reply:
Yup, thanks!